build-aux/check.mk

```diff
@@ -10,9 +10,44 @@ push-pytest-images: docker/kat-client.docker.push.remote
 push-pytest-images: docker/kat-server.docker.push.remote
 .PHONY: push-pytest-images
 
-# These are triggered by `python/tests/integration/manifests.py` (or by `push-pytest-images`)
+# test-{auth,ratelimit,shadow,stats}.docker
 test_svcs = auth ratelimit shadow stats
 $(foreach svc,$(test_svcs),docker/.test-$(svc).docker.stamp): docker/.%.docker.stamp: docker/%/Dockerfile FORCE
 	docker build --iidfile=$@ $(<D)
-$(foreach svc,$(test_svcs),docker/test-$(svc).docker): docker/%.docker: docker/.%.docker.stamp $(tools/copy-ifchanged)
-	$(tools/copy-ifchanged) $< $@
+
+# kat-client.docker
+docker/kat-client.go.layer.tar: $(tools/ocibuild) $(tools/write-ifchanged) FORCE
+	GOFLAGS=-mod=mod $(tools/ocibuild) layer gobuild ./cmd/kat-client | $(tools/write-ifchanged) $@
+docker/kat-client.fs.layer.tar: $(tools/ocibuild) $(tools/write-ifchanged) FORCE
+	{ $(tools/ocibuild) layer dir \
+	  --prefix=work \
+	  --chown-uid=0 --chown-uname=root \
+	  --chown-gid=0 --chown-uname=root \
+	  docker/kat-client; } | $(tools/write-ifchanged) $@
+docker/.kat-client.img.tar.stamp: $(tools/ocibuild) docker/base.img.tar docker/kat-client.go.layer.tar docker/kat-client.fs.layer.tar
+	{ $(tools/ocibuild) image build \
+	  --base=docker/base.img.tar \
+	  --config.Cmd='sleep' --config.Cmd='3600' \
+	  --tag=emissary.local/kat-client:latest \
+	  <($(tools/ocibuild) layer squash $(filter %.layer.tar,$^)); } > $@
```

Comment:
I have to say, this is a really rough idiom to read. Hmmm. Maybe `ocibuild image build` should be allowed to take a bunch of layer files on the command line and squash them.

Reply:
I disagree that this is hard to read. It sorta follow lispy indentation/brace style rather than C-ish indentation/brace style, which is I can respect as rough-if-unfamiliar, but I think the only problem is familiarity.

PS: web-dev folks may also be familiar-ish with this style because if you don't get to use a preprocessor like SASS or LESS, it's the most sane way to write CSS.